general-concepts/dependencies/text.xml

```diff
@@ -421,6 +421,35 @@ RDEPEND="media-libs/cogl:1.0=
 	gnutls? ( &gt;=net-libs/gnutls-2.8:= )"
 </codesample>
 
+<p>
+means that only the '1.0' slot is acceptable for <c>media-libs/cogl</c> and
+that sub-slot changes of <c>media-libs/cogl</c> will cause a rebuild of the
+dependent package. It furthermore means that every slot of
+<c>net-libs/gnutls</c> is acceptable but any slot change is causing a rebuild.
+</p>
+
+<p>
+The <c>:slot</c> dependency syntax continues to behave like in EAPI=4 or
+earlier, i.e., it indicates that only the specific slot value is acceptable and
```

Comment:
Not sure we need the comma after i.e., other than that, it looks fine to me.

Reply:
Yep, that is APA style, but I just remembered that devmanual uses BE. Will remove the comma.